src/openrct2/management/Award.cpp

```diff
@@ -273,18 +265,14 @@ static bool award_is_deserved_safest([[maybe_unused]] int32_t activeAwardTypes)
 /** All staff types, at least 20 staff, one staff per 32 peeps. */
 static bool award_is_deserved_best_staff(int32_t activeAwardTypes)
 {
-    uint16_t spriteIndex;
-    Peep* peep;
-    int32_t peepCount, staffCount;
-    int32_t staffTypeFlags;
-
     if (activeAwardTypes & (1 << PARK_AWARD_MOST_UNTIDY))
         return false;
 
-    peepCount = 0;
-    staffCount = 0;
-    staffTypeFlags = 0;
-    FOR_ALL_PEEPS (spriteIndex, peep)
+    auto peepCount = 0;
+    auto staffCount = 0;
+    auto staffTypeFlags = 0;
```

Comment:
These might not be receiving the type you expected them to. (I guess they are, just want to make sure you're aware)

Reply:
All fine for this one.